packages/app-studio-toolkit/test/public-api/create-workspace-proxy.spec.ts

```diff
@@ -19,6 +18,8 @@ describe("the `createWorkspaceProxy` utility", () => {
       onWorkspaceChanged(...args: any[]) {
         return args;
       },
+      startWatch() {},
```

Comment:
1. I don't think we should expose this methods.

Reply:
Yes, you are right.